README.md

```diff
@@ -242,3 +242,28 @@ The stop_delays view looks like this:
 ### Demo Project
 
 See the [gtfsrdb-delay-demo](https://github.com/CUTR-at-USF/gtfsrdb-delay-demo) project for a sample web application that visualizes these delays.
+
+### Scripts
+
+The scripts/ directory is provided as a "quick start" method for using GTFSrDB. A database server is still a prerequisite, and the scripts assume you're using PostgreSQL.
+
+To get started with the scripts, first make sure you have a good working Python environment. This involves setting up a virtual environment, then installing the required libraries. A quick way to do this is as follows:
+1. Ensure you are in the root directory of gtfsrdb on your PC.
+2. Create a Python virtual environment: `python3 -m venv venv`
+3. Install the required libraries: `pip3 install -r requirements.txt`
+
+Once your environment is configured, edit `env.sh` to ensure the `DB_HOST`, `DB_USER`, and `DB_PASS` are correct for your application. Also note that the `run.sh` script will create a database with the name specified by `DB`, so you may also alter this if you wish.
```

Comment:
Instead of a script I would suggest to use .env files. Are you aware of them? https://www.geeksforgeeks.org/how-to-create-and-use-env-files-in-python/ These would not need to be checked in and therefore there is less of a chance to commit some personal credentials